server/lib/tuist_web/marketing/components/marketing_layout_components/cta.html.heex

```diff
@@ -1,8 +1,8 @@
 <div class="marketing__component__cta">
   <h2 class="font-xxl-strong marketing__component__cta__title">
-    {gettext("Supercharge your Swift app development")}
+    {gettext("Unlock your iOS team's full potential")}
   </h2>
-  <.primary_button href={~p"/users/log_in"} size="medium">
-    {gettext("Get started")}
+  <.primary_button href="https://cal.tuist.dev/team/tuist/tuist" size="medium">
+    {gettext("Chat with the founders")}
```

Comment:
I would keep both CTAs – while I agree that we want to make the call link more prominent, getting folks to docs to get started is imho equally important. The new marketing page already has that (https://www.figma.com/design/2bUZjHvCbrh7dLfWR9EhyJ/Tuist-Website-2.0?node-id=54-4017&m=dev), so I'd preserve it here, too